From 022a86020081642839c377852a10978c4f816211 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?St=C3=A9phane=20Cerveau?= <scerveau@igalia.com>
Date: Wed, 31 May 2023 14:51:34 +0200
Subject: [PATCH 153/232] codecparsers: keep naming consistency in
 GST_H264_LEVEL

GST_H264_LEVEL_2 should be used instead of GST_H264_LEVEL_2_0

Part-of: <https://gitlab.freedesktop.org/gstreamer/gstreamer/-/merge_requests/4737>
---
 gst-libs/gst/codecparsers/gsth264parser.h | 2 +-
 gst-libs/gst/codecs/gsth264decoder.c      | 2 +-
 gst/codectimestamper/gsth264timestamper.c | 2 +-
 gst/videoparsers/gsth264parse.c           | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/gst-libs/gst/codecparsers/gsth264parser.h b/gst-libs/gst/codecparsers/gsth264parser.h
index f18a1f06b..e9e79e6d1 100644
--- a/gst-libs/gst/codecparsers/gsth264parser.h
+++ b/gst-libs/gst/codecparsers/gsth264parser.h
@@ -366,7 +366,7 @@ typedef enum
   GST_H264_LEVEL_L1_1 = 11,
   GST_H264_LEVEL_L1_2 = 12,
   GST_H264_LEVEL_L1_3 = 13,
-  GST_H264_LEVEL_L2_0 = 20,
+  GST_H264_LEVEL_L2 = 20,
   GST_H264_LEVEL_L2_1 = 21,
   GST_H264_LEVEL_L2_2 = 22,
   GST_H264_LEVEL_L3 = 30,
diff --git a/gst-libs/gst/codecs/gsth264decoder.c b/gst-libs/gst/codecs/gsth264decoder.c
index 15bd5dd2d..a30b98c2d 100644
--- a/gst-libs/gst/codecs/gsth264decoder.c
+++ b/gst-libs/gst/codecs/gsth264decoder.c
@@ -2278,7 +2278,7 @@ static const LevelLimits level_limits_map[] = {
   {GST_H264_LEVEL_L1_1, 3000, 396, 900, 192},
   {GST_H264_LEVEL_L1_2, 6000, 396, 2376, 384},
   {GST_H264_LEVEL_L1_3, 11800, 396, 2376, 768},
-  {GST_H264_LEVEL_L2_0, 11880, 396, 2376, 2000},
+  {GST_H264_LEVEL_L2, 11880, 396, 2376, 2000},
   {GST_H264_LEVEL_L2_1, 19800, 792, 4752, 4000},
   {GST_H264_LEVEL_L2_2, 20250, 1620, 8100, 4000},
   {GST_H264_LEVEL_L3, 40500, 1620, 8100, 10000},
diff --git a/gst/codectimestamper/gsth264timestamper.c b/gst/codectimestamper/gsth264timestamper.c
index 8e73dc2a0..420018705 100644
--- a/gst/codectimestamper/gsth264timestamper.c
+++ b/gst/codectimestamper/gsth264timestamper.c
@@ -210,7 +210,7 @@ static const LevelLimits level_limits_map[] = {
   {GST_H264_LEVEL_L1_1, 3000, 396, 900, 192},
   {GST_H264_LEVEL_L1_2, 6000, 396, 2376, 384},
   {GST_H264_LEVEL_L1_3, 11800, 396, 2376, 768},
-  {GST_H264_LEVEL_L2_0, 11880, 396, 2376, 2000},
+  {GST_H264_LEVEL_L2, 11880, 396, 2376, 2000},
   {GST_H264_LEVEL_L2_1, 19800, 792, 4752, 4000},
   {GST_H264_LEVEL_L2_2, 20250, 1620, 8100, 4000},
   {GST_H264_LEVEL_L3, 40500, 1620, 8100, 10000},
diff --git a/gst/videoparsers/gsth264parse.c b/gst/videoparsers/gsth264parse.c
index fb0145bbc..3a7c5520c 100644
--- a/gst/videoparsers/gsth264parse.c
+++ b/gst/videoparsers/gsth264parse.c
@@ -2054,7 +2054,7 @@ static const GstH264LevelLimit level_limits_map[] = {
   {GST_H264_LEVEL_L1_1, 768000},
   {GST_H264_LEVEL_L1_2, 1536000},
   {GST_H264_LEVEL_L1_3, 3041280},
-  {GST_H264_LEVEL_L2_0, 3041280},
+  {GST_H264_LEVEL_L2, 3041280},
   {GST_H264_LEVEL_L2_1, 5068800},
   {GST_H264_LEVEL_L2_2, 5184000},
   {GST_H264_LEVEL_L3, 10368000},
-- 
2.40.1

